index.html

```diff
@@ -802,9 +849,34 @@
     {name: "java"},
     {name: "cpp"},
     {name: "python"},
-    {name: "go", tabSize: 4, indentWithTabs: true, indentUnit: 4}
+    {name: "go", tabSize: 2, indentWithTabs: true, indentUnit: 4}
   ].forEach(spec => {
     editors[spec.name] = CodeMirror.fromTextArea(document.querySelector(`#code-${spec.name}`),
-                                                 Object.assign({mode: "google-" + spec.name}, spec))
+   Object.assign({
+    mode: "google-" + spec.name,
+    autoCloseBrackets: true,
+    continueComments: {key: "Enter", continueLineComment: false},
+    foldGutter: true,
+    gutters: ["gutter", "CodeMirror-linenumbers", "CodeMirror-foldgutter"],
+    height: "100%",
```

Comment:
There's no option named `height`

Reply:
heh, we've been setting it since time immemorial. perhaps it could be nice to throw some sort of error when unknown parameters are passed in? (this could be controlled by yet another parameter, e.g. `checkParameters`?)